scout/server/blueprints/cases/controllers.py

```diff
@@ -173,13 +160,24 @@ def case(store, institute_obj, case_obj):
     for panel_info in case_obj.get("panels", []):
         if not panel_info.get("is_default"):
             continue
-        panel_obj = store.gene_panel(
-            panel_info["panel_name"], version=panel_info.get("version")
-        )
+        panel_name = panel_info["panel_name"]
+        panel_version = panel_info.get("version")
+        panel_obj = store.gene_panel(panel_name, version=panel_version)
+        if not panel_obj:
+            LOG.warning(
+                "Could not fetch gene panel %s, version %s", panel_name, panel_version
+            )
+            LOG.info("Try to fetch latest existing version")
+            panel_obj = store.gene_panel(panel_name)
```

Comment:
Sure! I suppose one could also imagine the store having that, but that is just style. This is a new functionality, while not huge may be worth a little note somewhere?